src/screens/CreateCommunity/index.js

```diff
@@ -97,7 +97,7 @@ const CreateCommunity = () => {
     if (name && name.length >= MINIMUM_NAME_LENGTH && typeFetch === 'available') {
       const response = await submitCommunityName(name);
       if (response.success) {
-        navigation.replace('ContactScreen', {
+        navigation.replace('CreateCommunityCustomize', {
           isCreateCommunity: true,
           topicCommunityId: response.topic_id,
           topicCommunityName: name
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change seems to be replacing the navigation from 'ContactScreen' to 'CreateCommunityCustomize'. Ensure that all the necessary data is being passed to 'CreateCommunityCustomize' and it can handle all the props correctly. Also, make sure that this change doesn't break any existing functionality where 'ContactScreen' was expected after submitting a community name.
```diff
-        navigation.replace('ContactScreen', {
+        navigation.replace('CreateCommunityCustomize', {
           isCreateCommunity: true,
           topicCommunityId: response.topic_id,
           topicCommunityName: name
```
Also, consider adding error handling for the case when `submitCommunityName(name)` fails. Currently, if the submission fails, the user won't receive any feedback about the failure.


<!-- This is an auto-generated comment by OSS CodeRabbit -->